staging/src/k8s.io/legacy-cloud-providers/azure/azure_managedDiskController.go

```diff
@@ -236,6 +237,10 @@ func (c *ManagedDiskController) DeleteManagedDisk(diskURI string) error {
 
 	rerr = c.common.cloud.DisksClient.Delete(ctx, resourceGroup, diskName)
 	if rerr != nil {
+		if rerr.HTTPStatusCode == http.StatusNotFound {
+			klog.V(2).Infof("azureDisk - disk(%s) is already deleted", diskURI)
```

Comment:
thanks for the catch.
`V(2)` is actually what I want since it would be printed in our log level setting in the service. This is like a warning msg that underneath disk is already deleted, so maybe warning is more appropriate.